kmodes/kprototypes.py

```diff
@@ -130,13 +130,17 @@ def __init__(self, n_clusters=8, max_iter=100, num_dissim=euclidean_dissim,
                       "Setting n_init to 1.")
             self.n_init = 1
 
-    def fit(self, X, y=None, categorical=None):
+    def fit(self, X, y=None, categorical=None, sample_weights=None):
         """Compute k-prototypes clustering.
 
         Parameters
         ----------
         X : array-like, shape=[n_samples, n_features]
         categorical : Index of columns that contain categorical data
+        sample_weights : sequence, default: None
```

Comment:
nit: add newline between parameters

Reply:
Let's name this parameter `sample_weight` instead, because this is what scikit-learn uses. (E.g., https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#sklearn.cluster.KMeans.fit)

Reply:
Excellent point!